spec/injection.spec.ts

```diff
@@ -1030,7 +1124,7 @@ describe('Injector', () => {
     }
 
     describe('Scoped injection', () => {
-        const testCount = 30;
+        const testCount = 10;
```

Comment:
what is testCount used for?

Reply:
This controls the number of depth tests.  It also controls how deep the depth test is.  I felt 30 was too much so reduced a little.